sale_order_line_price_history/wizards/sale_order_line_price_history.py

```diff
@@ -12,12 +12,19 @@ def _default_partner_id(self):
         line_id = self.env.context.get("active_id")
         return self.env["sale.order.line"].browse(line_id).order_partner_id.id
 
+    @api.model
+    def _default_product_id(self):
+        line_id = self.env.context.get("active_id")
+        return self.env["sale.order.line"].browse(line_id).product_id.id
```

Comment:
Why do you need this if you are already passing `default_product_id` in the context?

Reply:
This was a related field. I preferred to keep the ability to call the wizard with just the line id in the context